utils

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
```

Comment:
Looks like the changes to this file inadvertently broke Travis on ome/openmicroscopy - see e.g. https://travis-ci.org/ome/openmicroscopy/builds/618399019

```
+ cd /src

/infra/py-common: line 10: cd: /src: No such file or directory

Stopping openmicroscopy_db_1_5cff548c8a94    ... 

Stopping openmicroscopy_omero_1_47ce18febf14 ... 

Stopping openmicroscopy_test_1_bdd866aa92d6  ... 

oving openmicroscopy_db_1_5cff548c8a94    ... 

Removing openmicroscopy_omero_1_47ce18febf14 ... 

Removing openmicroscopy_test_1_bdd866aa92d6  ... 

oving network openmicroscopy_default

The command ".omero/docker srv" exited with 1.
```

It looks like the `srv` phase has been silently failing for a while but this makes any error fatal to the entire build